include/rocksdb/advanced_options.h

```diff
@@ -62,13 +62,6 @@ struct CompactionOptionsFIFO {
   // Default: 1GB
   uint64_t max_table_files_size;
 
-  // Drop files older than TTL. TTL based deletion will take precedence over
-  // size based deletion if ttl > 0.
-  // delete if sst_file_creation_time < (current_time - ttl)
-  // unit: seconds. Ex: 1 day = 1 * 24 * 60 * 60
-  // Default: 0 (disabled)
-  uint64_t ttl = 0;
```

Comment:
Now that we are removing this option from inside FIFO options, lets add a few lines above the ttl option on line 632 to say that it can be used for FIFO as well (may be copy paste these above lines).